### DISCUSSION TITLE
[17주차] 온라인 스터디

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#113] [General]
~ 20:15 각자 PR리뷰
~ 21:00 발표
~ 21:10 회고 (좋았던 점, 건의사항, 느낀점 등)

**Comment by ChaerinYu** [comment]
### 발표
1. 유채린
* [JPA와 Querydsl] querydsl bulk연산 #109
2. 김정윤
* [JAVA] LinkedList와 ArrayList 차이 #111
3. 이주형
* [Network] SSL HandShake #114

**Reply by KJY97** [reply]
오늘은 몰랐던 것들을 새롭게 아는 날이었습니다.. 감사합니다!

**Reply by ChaerinYu** [reply]
오랜만에 참석했네요 ㅎㅎ,, ... 고생 많으셨습니다~~

**Reply by leeejuhyeong** [reply]
준비 미흡...! 다시 정신차리고오겠습니다.
